modules/backend/controllers/Auth.php

```diff
@@ -32,6 +32,8 @@ class Auth extends Controller
     public function __construct()
     {
         parent::__construct();
+        // Add JS File to un-install SW to avoid Cookie Cache Issues when Signin, see github issue: #3707
+        $this->addJs('~/../../../../modules/backend/assets/js/auth/uninstall-sw.js');
```

Comment:
why is this so excessive with the `../` directory links?

Reply:
It doesn't work without them ?

Reply:
Screenshot on my test machine:

![eg](https://user-images.githubusercontent.com/17784082/52087383-2c050680-25a1-11e9-9244-baf49f9412e4.png)

Reply:
This:

```html
~/modules/backend/assets/js/auth/uninstall-sw.js
```

Gives me this:

```html
http://localhost/admincp/backend/auth/~/modules/backend/assets/js/auth/uninstall-sw.js
```

Does that mean something is wrong?

Reply:
hmm, no it's just not parsing it. Could you just use the `url()` function instead?